aries-site/src/pages/design-tokens/how-to-read-design-tokens.mdx

```diff
@@ -0,0 +1,100 @@
+import { Example } from '../../layouts';
+import { Box, Text } from 'grommet';
+import { BetaNotification, TokenHighlight } from '../../examples';
+
+<BetaNotification />
+
+<Example height={{ min: 'xsmall' }}>
+  <Text size="xlarge">hpe.button.primary.enabled.background</Text>
+</Example>
+
+A token is read in a left-to-right order, moving from general to specific.
+
+The structure of the token naming schema intends to:
+
+1. Establish the subject of the token (a color, a dimension such as spacing, a component such as button).
+2. Specify the variant and state of the subject.
+3. In the case of component tokens, identify the CSS property to be affected.
+
+## Naming convention
+
+A consistent convention makes it easier to understand what tokens do, how they should be used, and how new tokens could be added in the future.
+
+These are all of the possible parts of a token name, but each [type of design token](#types-of-design-tokens) uses a subset of these.
+
+<Example height="auto">
+  <TokenHighlight highlight="all" />
+</Example>
+
+| Category  | Definition                                                                                                                                                                                                         | Examples                                          | Notes                                                                                                                                                        |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| prefix    | Provides top-level encapsulation of a particular flavor of HPE theme. The prefix helps to avoid collisions and improve developer experience when scanning styles to identify those belonging to HPE Design Tokens. | hpe                                               | --                                                                                                                                                           |
```

Comment:
Should we be treating the primary column with a bolder font weight?

```suggestion
| prefix    | Provides top-level encapsulation of a particular flavor of HPE theme. The prefix helps to avoid collisions and improve developer experience when scanning styles to identify those belonging to HPE Design Tokens. | hpe                                               | --                                                                                                                                                           |
```

Reply:
Done.